lib/sshkit/version.rb

```diff
@@ -1,3 +1,3 @@
 module SSHKit
-  VERSION = "1.16.1".freeze
+  VERSION = "1.17.0".freeze
```

Comment:
Sorry, only maintainers are allowed to change the version 😅

Reply:
No problemo!